### PR TITLE
fix(a11y): update element to login for github and google

### PR DESCRIPTION
**Pull request by jyash97**
## Pull request type

Please check the type of change your PR introduces:

- [x] Bugfix

## Pull request checklist

Please check if your PR fulfills the following requirements:

- [x] If it's a frontend change, Prettier has been run

## What is the current behavior?
Currently, the OAuth buttons are `div` which cant be accessed with the keyboard, and the screen reader cant differentiate it as a button.

## What is the new behavior?
Update element to be button

## Screenshot
image

### CODE REVIEW

**Comment by jnfrati** [comment]
Hey @jyash97! Thank you so much for your contribution! let me know when it's ready to review!

**Comment by jyash97** [comment]
Hey @jnfrati 👋 its ready for review

File: dashboard/src/main/auth/Login.tsx [hunk 1]
```diff
@@ -146,7 +146,7 @@ export default class Login extends Component<PropsType, StateType> {
   };
 
   renderGoogleSection = () => {
-    if (this.state.hasGoogle) {
+    if (true || this.state.hasGoogle) {
```
Comment (jnfrati) [inline dashboard/src/main/auth/Login.tsx:149]:
```suggestion
    if (this.state.hasGoogle) {
```
Everything looks awesome! Just one small change, can we remove this true?

Comment (jyash97) [reply dashboard/src/main/auth/Login.tsx:149]: So sorry I missed this. It was just for testing locally. Thanks a lot for catching this

**Review by jnfrati — APPROVED** [review]
Thanks for the contribution @jyash97! Any other suggestions or improvements to make Porter accessible are super welcomed!

**Pull request merged**